### PR TITLE
more updates

### LINKED ISSUE #431: combine ifxy.f adn30.f idn30.f igetfxy.f into one file fxy.F90?
How would it be to combine ifxy.f adn30.f idn30.f igetfxy.f into one file fxy.F90?

### PR DESCRIPTION
**Pull request by jbathegit**
Fixes #431

Part of #254
Part of #328

### CODE REVIEW

File: src/cktaba.f [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
-C> @param[in] LUN - integer: file ID.
+C> @param[in] LUN - integer: file ID
 C> @param[out] SUBSET - character*8: Table A mnemonic
 C>                      - returned as a string of all blank characters
 C>                        if IRET is equal to 11 (see below) and if Section 3
 C>                        isn't being used for decoding
 C> @param[out] JDATE    - integer: date-time stored within Section 1 of BUFR
 C>                        in format of either YYMMDDHH or
-C>                        YYYYMMDDHH, depending on datelen() value.
+C>                        YYYYMMDDHH, depending on datelen() value
```
Comment (edwardhartnett) [inline src/cktaba.f:18]: How come you are taking away the punctuation?

**Review by edwardhartnett — APPROVED** [review]
It's good to see this consolidation into files take place. I've also been doing this in g2 and g2c, and it helps...

File: src/cktaba.f (hunk 1 above)
Comment (jbathegit) [reply src/cktaba.f:18]: B/C some time ago you yourself had pointed out that these variable descriptions usually aren't complete sentences and therefore don't really need periods after them :-)

Comment (edwardhartnett) [reply src/cktaba.f:18]: OK, just curious.

**Pull request merged**